eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -192,7 +200,10 @@ object NodeParams {
       toRemoteDelayBlocks = config.getInt("to-remote-delay-blocks"),
       maxToLocalDelayBlocks = config.getInt("max-to-local-delay-blocks"),
       minDepthBlocks = config.getInt("mindepth-blocks"),
-      smartfeeNBlocks = 3,
+      feeEstimator = new FeeEstimator {
```

Comment:
fee estimator should be provided to `makeNodeParams`, not defined here

Reply:
Addressed in f0649dd